assets/src/edit-story/components/canvas/useInsertElement.js

```diff
@@ -83,6 +83,11 @@ function useInsertElement() {
           }
         }, 0);
       }
+      // When done, send fake focusout event to document to force focus to new element
+      setTimeout(() => {
```

Comment:
As much as I don't like setTimeout, but IMHO we won't get away from some ugliness with focus. My only request is to put this into a separate hook: `canvas/useFocusCanvas` and just call it here.

Reply:
Makes sense. Done in 33c21ddc and ef683a97.